### PR TITLE
Bump Spring Cloud GCP version to 1.2

### PR DESCRIPTION
**Pull request by meltsufin**

### CODE REVIEW

**Comment by kurtisvg** [comment]
@dzlier-gcp - Any insight into why this version bumps aren't being performed by renovate?

**Comment by elharo** [comment]
At least one of the dependencies doesn't seem to follow anything approximating semver; e.g. HOXTON.release

File: vision/spring-framework/pom.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@ limitations under the License.
 			<dependency>
```
Comment (averikitsch) [inline vision/spring-framework/pom.xml]: I know the original formatting is off, but would you be able to realign this pom file?

Comment (meltsufin) [reply vision/spring-framework/pom.xml]: How is it off? Is this about tabs vs. spaces?

**Comment by meltsufin** [comment]
> At least one of the dependencies doesn't seem to follow anything approximating semver; e.g. HOXTON.release

@elharo Hoxton is a release train name in Spring Cloud and they use it as version for the BOM. We don't really have control over it.

File: vision/spring-framework/pom.xml (hunk 1 above)
Comment (averikitsch) [reply vision/spring-framework/pom.xml]: It's using tabs not spaces, but just a nit since it renders differently on github.

Comment (meltsufin) [reply vision/spring-framework/pom.xml]: Done.

**Pull request merged**